chat-components/src/components/notificationpane/interfaces/chatdisconnect/IChatDisconnectStyleProps.ts

```diff
@@ -0,0 +1,16 @@
+import { IStyle } from "@fluentui/react";
+import { INotificationPaneStyleProps } from "../common/INotificationPaneStyleProps";
+import { IChatDisconnectClassNames } from "./IChatDisconnectClassNames";
+
+/**
+ * This interface will have all the style properties as customized by C1 for chat disconnect scenario
+ * It extends the common style properties <INotificationPaneStyleProps>
+ */
+export interface IChatDisconnectStyleProps extends INotificationPaneStyleProps {
```

Comment:
Where are all the default styleProps stored?

Reply:
default styleProps are stored in chat-widget, and merged within `NotificationPaneStateful`. See https://github.com/microsoft/omnichannel-chat-widget/pull/351/files#diff-baa95c5389a3f9b9ffd90dccce69c94a784a12d41fdd3b6cdbeffd68c1938708 as an example.

Reply:
Will this cause an issue if a customer only wants to use chat-component but not chat-widget? What happens when this component is used as is?